lib/process/processWrapper.js

```diff
@@ -21,6 +21,7 @@ class ProcessWrapper {
    */
   constructor(_options) {
     this.interceptLogs();
+    this.env = _options ? _options.env : 'development';
```

Comment:
Why do you need env here too?

Reply:
Because not all of the processes have their constructors invoked with an options object. In that case, it falls back to `'development'`, the same way we do with the cli when no environment is specified.

Reply:
ProcessorWrapper instances are instantiated when a message is sent to the child process. Upon further review, it is necessary to have the line you commented on here in order for the env property to make it to the wrapper instance in webpackProcess.js. However, I determined it is not necessary to also put the env property on a ProcessLauncher instance.

Reply:
Anyway, I decided to be more conservative:  I've made a change so that only the WebpackProcess subclass of ProcessWrapper is setting `this.env` from `options.env`.